TeamSnapSDK/SDK/DataTypes/Lineups/TSDKEventLineup.m

```diff
@@ -28,7 +28,8 @@ + (void)updateEventLineup:(TSDKEventLineup *)lineup withLineupEntries:(NSArray <
         NSMutableArray *lineupProperties = [NSMutableArray array];
         
         for (NSString *key in @[@"member_id", @"sequence", @"label"]) {
-            NSString *value = [lineupEntry.collection.data valueForKey:key];
+            
+            NSString *value = [lineupEntry collectionObjectForKey:key];
```

Comment:
should this be `getString` to be more explicit?

Reply:
Fixed